specs/atom_packages.table

```diff
@@ -7,7 +7,10 @@ schema([
     Column("path", TEXT, "Package's package.json path"),
     Column("license", TEXT, "License for package"),
     Column("homepage", TEXT, "Package supplied homepage"),
+    Column("uid", BIGINT, "The local user that owns the plugin",
```

Comment:
Is this required?

Reply:
Very much so, since `atom_packages` are installed per-user. The `user_data` attribute requires either `uid` or `username`.

Reply:
Does it need `required=True`?

Reply:
Nope, it's an optional input. Meaning it will take a different action if the constraint exists. The `additional` keyword applies this checking/preference in the index selection.